clerk/organizations.go

```diff
@@ -66,6 +69,44 @@ func (s *OrganizationsService) Update(organizationID string, params UpdateOrgani
 	return &organization, nil
 }
 
+type UpdateOrganizationLogoParams struct {
+	File           multipart.File
+	Filename       string
```

Comment:
I couldn't find an elegant way to get the filename from the file. If anybody has any suggestion, please help. :smile:

Reply:
Made the parameter optional.